webpack/browserConfig.js

```diff
@@ -10,7 +10,6 @@ const path = require('path');
 const webpack = require('webpack');
 
 const definePluginConfig = require('./definePluginConfig');
-const dirs = require('./dirs');
```

Comment:
Nothing in https://github.com/metabrainz/musicbrainz-server/commit/1c9f19ef16fdc2faf09fc59f0bf2255a0ac78019#diff-193049bfe9b64e7e292905faec96d3acdf2f926ae16f4ca87cfc37dcd23d2738R13 suggests it has ever been needed.